documentation/content/Components/photoswipe-instagramp-gallery.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: Gallery -- Use Instagram
+authors: Talha Mansoor, Pablo Iranzo Gómez
+Tags: nuances, images, gallery, instagram
+Date: 2020-02-06
+Slug: photoswipe-gallery-using-instagram
+Category: Components
+---
+
+Similar to the use case defined in [Photogallery]({filename}photogallery.md), Pelican has code in place for showing instragram galleries.
+
+## Article contents
+
+To enable the plugin, just define a similar div inside your article for each gallery:
```

Comment:
To embed Instagram gallery, just define a `div` in this manner,

Reply:
done